exports/taskfiles/utils/cmake.yaml

```diff
@@ -174,6 +176,8 @@ tasks:
         {{default "" .CMAKE_SETTINGS_DIR}}
       CMAKE_TARGETS:
         ref: "default (list) .CMAKE_TARGETS"
+      CMAKE_SOURCE_DIR: >-
```

Comment:
With the addition of this variable, the `SOURCE_DIR` variable is now confusingly named. How about changing it to `EXTRACTION_DIR`? If so, can we also change:

* `SOURCE_DIR`'s default's suffix from `-src` to `-extracted`?
* `WORK_DIR`'s description from "source directories" to "extraction directories"?